### DISCUSSION TITLE
Where's setTetherNode in osgEarth2.10？

### DISCUSSION THREAD

**Discussion by MRdesperado1** [pelicanmapping/osgearth#1839] [Support Q&A] [answered]
Hello, I found no setTetherNode  function when using osgEarth2.10. What should I do to bind a node?
How can I bind the viewpoint to the aircraft model so that the viewpoint and the model move at the same time?
I use OSG version 3.6.3 and osgearth version 2.10.
In addition, when I try to paste an airport model on my osgearth, my airport model is always suspended, even if I set the altitude to 0.
![1](https://user-images.githubusercontent.com/38163799/138323864-7c9b233c-c5d8-4c7a-8e9d-becb6357af9d.png)

The code is as follows：
        
       csn = new osg::CoordinateSystemNode;
	csn->setEllipsoidModel(new osg::EllipsoidModel);
	airport = osgDB::readNodeFile("E:/OSG/Data/model/airport.ive" );
	mtAirport = new osg::MatrixTransform;
	/*mtAirport->getOrCreateStateSet()->setMode(GL_RESCALE_NORMAL, osg::StateAttribute::ON);*/
	mtAirport->getOrCreateStateSet()->setMode(GL_LIGHTING, osg::StateAttribute::OFF | osg::StateAttribute::OVERRIDE);
	mtAirport->addChild(airport);
	mRoot->addChild(mtAirport);

	osg::Matrixd mtTemp;
	csn->getEllipsoidModel()->computeLocalToWorldTransformFromLatLongHeight(osg::DegreesToRadians(34.3762), osg::DegreesToRadians(109.1263), 0,mtTemp);
	mtAirport->setMatrix(mtTemp);

Thank you very much!!!

**Comment by gwaldron** [comment, marked as answer]
To lock the camera on a node, create an `osgEarth::Viewpoint` and call `setNode`. Then call `EarthManipulator::setViewpoint` to apply the viewpoint.

**Reply by MRdesperado1** [reply]
Thank you very much. I still want to ask you. My airport displays floating in the air. As the viewpoint moves, the airport will also move. How should this situation be solved?
Thanks again.

**Reply by gwaldron** [reply]
There is not enough information to answer this question -- also, please start a new thread for a different topic.

**Reply by MRdesperado1** [reply]
ok, thank you very much.
